langchain/src/retrievers/parent_document.ts

```diff
@@ -73,9 +67,9 @@ export class ParentDocumentRetriever extends BaseRetriever {
     }
     const parentDocs: Document[] = [];
     for (const parentDocId of parentDocIds) {
-      const parentDoc = await this.docstore.search(parentDocId);
-      if (parentDoc !== undefined) {
-        parentDocs.push(parentDoc);
+      const storedParentDocs = await this.docstore.mget([parentDocId]);
```

Comment:
can we use `mget(parentDocIds)` since it's the optimized variant

Reply:
Ah right